cli/flox-rust-sdk/src/models/environment/mod.rs

```diff
@@ -711,9 +719,62 @@ pub fn path_hash(p: &impl AsRef<Path>) -> String {
     chars.to_string()
 }
 
+/// Return a path to the services socket given a unique identifier
```

Comment:
Blocker: I think this comment needs to be updated. I don't see us actually using `/tmp` and it doesn't mention using `~/.cache/flox` instead.

Reply:
Whoops updated